code/game/objects/gamemode/crash/items.dm

```diff
@@ -0,0 +1,206 @@
+			has_auth = TRUE
+
+
+/obj/machinery/nuclearbomb/crash/attack_hand(mob/living/user)
+	. = ..()
+	if(.)
+		return
+	if (extended)
+		if (!ishuman(user))
+			to_chat(user, "<span class='warning'>You don't have the dexterity to do this!</span>")
+			return TRUE
+		user.set_interaction(src)
+
+
+		var/safe_text = (safety) ? "Safe" : "Engaged"
+		var/status
+		if (has_auth)
+			if (timing)
+				status = "Func/Set-[safe_text]"
+			else
+				status = "Functional-[safe_text]"
+		else
+			if (timing)
+				status = "Set-[safe_text]"
+			else
+				status = "Auth. S1-[safe_text]"
+		
+		var/html = {"
+		<b>Nuclear Fission Explosive</b><br />
+		<hr />
+		<b>Status</b>: [status]<br />
+		<b>Timer</b>: [timeleft]<br />
+		<br />
+		Timer: [timing ? "On" : "Off"] [has_auth ? "<a href='?src=[REF(src)];timer=1'>Toggle</a>" : "Toggle"] <br />
+		Time: [has_auth ? "<a href='?src=[REF(src)];time=-10'>--</a> <a href='?src=[REF(src)];time=-1'>-</a> [timeleft] <a href='?src=[REF(src)];time=1'>+</a> <a href='?src=[REF(src)];time=10'>++</a>" : "- - [timeleft] + +"] <br />
+		<br />
+		Safety: [safety ? "On" : "Off"] [has_auth ? "<a href='?src=[REF(src)];safety=1'>Toggle</a>" : "Toggle"] <br />
+		Anchor: [anchored ? "Engaged" : "Off"] [has_auth ? "<a href='?src=[REF(src)];anchor=1'>Toggle</a>" : "Toggle"] <br />
+		<hr />
+		Red Auth. Disk: <a href='?src=[REF(src)];r_auth=1'>[r_auth ? "++++++++++" : "----------"]</a><br />
+		Green Auth. Disk: <a href='?src=[REF(src)];g_auth=1'>[g_auth ? "++++++++++" : "----------"]</a><br />
+		Blue Auth. Disk: <a href='?src=[REF(src)];b_auth=1'>[b_auth ? "++++++++++" : "----------"]</a><br />
+		"}
+
+		var/datum/browser/popup = new(user, "nuclearbomb", "<div align='center'>Nuclear Bomb</div>", 300, 400)
+		popup.set_content(html)
+		popup.open(FALSE)
+		onclose(user, "nuclearbomb")
+	else if (deployable)
+		if (!do_after(user, 3 SECONDS, TRUE, src, BUSY_ICON_BUILD))
+			return
+		if(removal_stage < NUKE_STAGE_BOLTS_REMOVED)
+			anchored = TRUE
+			visible_message("<span class='warning'>With a steely snap, bolts slide out of [src] and anchor it to the flooring!</span>")
+		else
+			visible_message("<span class='warning'>\The [src] makes a highly unpleasant crunching noise. It looks like the anchoring bolts have been cut.</span>")
+		if(!lighthack)
+			flick("nuclearbombc", src)
+			icon_state = "nuclearbomb1"
+		extended = TRUE
+
+
+/obj/machinery/nuclearbomb/crash/Topic(href, href_list)
+	. = ..()
+	if(.)
+		return
+
+	usr.set_interaction(src)
+	
+	if(href_list["r_auth"])
+		if (r_auth)
+			r_auth.loc = loc
+			has_auth = FALSE
+			r_auth = null
+		else
+			var/obj/item/I = usr.get_active_held_item()
+			if (istype(I, /obj/item/disk/nuclear/crash/red))
+				if(usr.drop_held_item())
+					I.forceMove(src)
+					r_auth = I
+	if(href_list["g_auth"])
+		if (g_auth)
+			g_auth.loc = loc
+			has_auth = FALSE
+			g_auth = null
+		else
+			var/obj/item/I = usr.get_active_held_item()
+			if (istype(I, /obj/item/disk/nuclear/crash/green))
```

Comment:
Can make it into an elseif and combine those two lines with the proc call and istype

Reply:
I don't quite understand this request, could you give me an example?